src/components/screens/error-page.tsx

```diff
@@ -7,16 +7,14 @@ import { useRouteError } from "react-router-dom";
  */
 function ErrorPage () {
   const error: unknown = useRouteError();
-  console.error(error);
 
   return (
     <div id="error-page">
       <h1>Oops!</h1>
       <p>Sorry, an unexpected error has occurred.</p>
       <p>
         <i>
-          {error.statusText}
-          {error.message}
+          {(error as Error)?.message || (error as { statusText?: string })?.statusText}
```

Comment:
What's the point of this `error-page`?

Reply:
> What's the point of this `error-page`?
I made an error-page to be an error element because react-router-dom official docs recommended using one. However, I briefly discussed this with Eetu this morning, as to why oss-survey-management app doesn't use one, apparently there's no point in it.

Reply:
> What's the point of this `error-page`?

I made an error-page to be an error element because react-router-dom official docs recommended using one. However, I briefly discussed this with Eetu this morning, as to why oss-survey-management app doesn't use one, apparently there's no point in it.

Reply:
For now, the error-page exists to be displayed if the accessed route/path does not exist. Hopefully it also could be configured to displayed if the logged user is unauthorized to access said route. 

We were discussing about the necessity of this page as opposed to using dialog-based error handling for example. I'll leave it as it is for now.